styleguide/src/components/ElapsedDuration.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Typography } from '@material-ui/core'
+import { elapsedDuration } from '../utils/elapsedDuration'
+
+interface Props {
+  start: string
+  end: string
+  className?: string
+}
+
+export const ElapsedDuration: React.FC<Props> = ({ start, end, className }) => {
+  return (
+    <Typography role="elapsedduration" variant="h6" className={className}>
```

Comment:
Should this be camel case?

Reply:
My understanding is that ARIA roles should be lowercase combined words. e.g. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Rowgroup_Role

I switched it to a role from an id so that there can be multiple on the same page FWIW.